ui/pages/confirmations/components/signature-request-original/signature-request-original.component.js

```diff
@@ -18,29 +18,26 @@ import {
 } from '../../../../helpers/utils/util';
 import { isSuspiciousResponse } from '../../../../../shared/modules/security-provider.utils';
 import SiteOrigin from '../../../../components/ui/site-origin';
-import Typography from '../../../../components/ui/typography/typography';
 import { PageContainerFooter } from '../../../../components/ui/page-container';
 import {
-  TypographyVariant,
-  FontWeight,
+  TextVariant,
   TextAlign,
   TextColor,
   Size,
   ///: BEGIN:ONLY_INCLUDE_IF(build-mmi)
   IconColor,
   Display,
   BlockSize,
-  TextVariant,
   BackgroundColor,
   ///: END:ONLY_INCLUDE_IF
 } from '../../../../helpers/constants/design-system';
 import {
   ButtonLink,
+  Text,
   ///: BEGIN:ONLY_INCLUDE_IF(build-mmi)
   Box,
   Icon,
   IconName,
-  Text,
```

Comment:
We need to make sure that Devin respects these environment tags so it should not rearrange the imports here